.github/workflows/code-quality.yml

```
@@ -3,7 +3,7 @@ name: Code Quality
 on:
   push:
     branches:
-      - '*'
+      - 'main'
```

так как-то даже поприятнее
```suggestion
      - main
```